runtime/compiler/optimizer/SPMDParallelizer.cpp

```diff
@@ -1369,6 +1369,29 @@ bool TR_SPMDKernelParallelizer::reductionLoopExitProcessing(TR::Compilation *com
    return true;
    }
 
+void TR_SPMDKernelParallelizer::replaceAndAnchorOldNode(TR::Compilation *comp, TR::TreeTop *treeTop, TR::Node *parent, TR::Node *oldNode, TR::Node *newNode, int index)
+   {
+	   treeTop->insertBefore(TR::TreeTop::create(comp, TR::Node::create(TR::treetop, 1, oldNode)));
+	   oldNode->recursivelyDecReferenceCount();
+	   parent->setAndIncChild(index, newNode);
```

Comment:
There are still some tabs here that should be replaced with spaces.

Reply:
Fixed